### PR TITLE
Improve SEO of page

### PR DESCRIPTION
**Pull request by christian-bromann**
Improving SEO of blog posts based on Pallavis suggestions.

### CODE REVIEW

File: content/projects/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Projects"
 date: 2019-05-12T12:14:34+06:00
-description: "This is meta description."
+description: "Currated Open Source projects maintained by Sauce Labs and the Open Source Program Office."
```
Comment (diemol) [inline content/projects/_index.md]:
```suggestion
description: "Curated Open Source projects maintained by Sauce Labs and the Open Source Program Office."
```

**Review by diemol — CHANGES_REQUESTED** [review]

**Review by diemol — APPROVED** [review]

File: layouts/partials/head.html [hunk 2]
```diff
@@ -52,4 +52,50 @@
     navigator.serviceWorker.register('/sw.js', { scope: '/' })
   }
   </script>
+  {{ if .IsPage }}
```
Comment (diemol) [inline layouts/partials/head.html:55]: I don't think all this is needed, Hugo has a embedded template for it
https://www.continuoustestingmeetup.com/events/2019-05-23/

**Review by diemol — CHANGES_REQUESTED** [review]

File: layouts/partials/head.html (hunk 2 above)
Comment (diemol) [reply layouts/partials/head.html:55]: I just tried it on the CTM site and it works
https://github.com/continuoustestingmeetup/continuoustestingmeetup.github.io/commit/dc6468d3e954cc7c9ddab66494039518415ac8ee

Comment (christian-bromann) [reply layouts/partials/head.html:55]: Good catch, updated!

File: layouts/partials/head.html [hunk 3]
```diff
@@ -52,4 +52,46 @@
...
+  <meta name="twitter:domain" value="opensource.saucelabs.com" />
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:title" value="{{ .Title }}" />
+  {{ if isset .Params "summary" }}
+    <meta name="twitter:description" value="{{ .Params.summary }}"/>
+  {{ end }}
+
+  {{ if isset .Params "image" }}
+    <meta property="og:image"  content="{{.Params.image | absURL}}">
+    <meta property="twitter:image"  content="{{.Params.image | absURL}}">
```
Comment (diemol) [inline layouts/partials/head.html]: Isn't this, duplicating things already present in the `opengraph.html` [template](https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/opengraph.html#L6)?

File: layouts/partials/head.html [hunk 4]
```diff
@@ -52,4 +52,46 @@
...
+    "wordCount" : "{{ .WordCount }}",
+    {{ if .Params.image }}"image" : "{{ .Params.image | absURL }}",{{ end }}
+    "keywords" : [ "{{ range $i, $e := .Params.tags }}{{ if $i }}", "{{ end }}{{ $e }}{{ end }}" ]
+  }
+  </script>
+
+  <!-- Open Graph Metadata -->
+  {{ template "_internal/opengraph.html" . }}
+
+  <meta name="twitter:domain" value="opensource.saucelabs.com" />
```
Comment (diemol) [inline layouts/partials/head.html]: There is a Twitter [template](https://gohugo.io/templates/internal/#twitter-cards) as well, I thought I had mentioned it before. Not sure if all the attributes are there but I see `twitter:card` is [there](https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/twitter_cards.html#L2), same as above, we should not duplicate this logic.

**Review by diemol — CHANGES_REQUESTED** [review]

**Comment by christian-bromann** [comment]
@diemol that's neat! I updated the branch.

**Review by diemol — APPROVED** [review]
🎉

**Pull request merged**